### DISCUSSION TITLE
I uploaded all the information as required. Is it because the file is in Chinese so it doesn’t pass? discrimination?

### DISCUSSION THREAD

**Discussion by bruce0781** [community/community#53679] [GitHub Education] [answered]
I am a student from Zhongguancun High School in Beijing, my school may not be well-known, and I am from Beijing, China, so all my official documents or student card or admission letter are in Chinese, I don't know if it is because For this reason, Github failed my application 5 times.
Through the benefits of Github, I can run the programs and projects I made very well, but unfortunately I can't manage and connect well with SSH now.

**Comment by samus-aran** [comment, marked as answer]
Hi @Bruce081 

Here is a quick tip on submitting non english evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying. 👈 

Let us know how you get on 👍

**Reply by bruce0781** [reply]
Do you mean that my username needs to be the same as the name on my certificate?
